BAZProject/HomeMovieViper/View/HomeViewCollectionsExtension.swift

```diff
@@ -44,6 +44,13 @@ extension HomeMoviesView: UICollectionViewDataSource{
 // MARK: - CollectionView's Delegate
 
 extension HomeMoviesView: UICollectionViewDelegate{
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        if collectionView == self.moviesCollectionView{
```

Comment:
What's the intention of this validation? Is it possible to have more collections having the same delegate? If so, the best way to address that is to have different classes for each delegate needed. Otherwise these methods would grow with the project scope